YACReaderLibrary/library_window.cpp

```diff
@@ -1798,8 +1798,6 @@ void LibraryWindow::checkEmptyFolder()
 void LibraryWindow::openComic(const ComicDB &comic)
 {
     if(!importedCovers) {
-        QList<ComicDB> siblings = comicsModel->getAllComics();
```

Comment:
Good catch, this is now done in YACReader, it request the siblings to the local server in YACReaderLibrary.